src/components/modals/baseModal.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <Teleport to="#modals">
+    <div v-if="show" class="">
```

Comment:
Is show here and in configmationDialog really needed? Isn't enough to put a v-if on the component tag?